nanocloud/models/users/users.go

```diff
@@ -105,9 +107,9 @@ func FindUsers() ([]*User, error) {
 			&user.Email,
 			&user.IsAdmin,
 			&user.Activated,
-			&user.SignupDate,
+			&timestamp,
 		)
-		user.SignupDate *= 1000
+		user.SignupDate = int(1000 * timestamp)
```

Comment:
Magic value here.
I suggest you to put a comment on this line to indicate why you multiply by a thousand your signupDate